stores/peerstore/src/peerstore.rs

```diff
@@ -99,7 +99,6 @@ pub trait PeerAccess {
     /// Sets the TTL of an address of a peer. Adds the address if it is currently unknown.
     ///
     /// Contrary to `add_addr`, this operation is never a no-op.
-    #[inline]
```

Comment:
Was this a clippy error?

Reply:
https://rust-lang.github.io/rust-clippy/list/index.html#inline_fn_without_body